### PR TITLE
hotfix: noop callbacks for the createAndSave function solves #171

### PR DESCRIPTION
**Pull request by kyuumeitai**
Dirty but solves the issue.

### CODE REVIEW

File: bin/cleaver [hunk 1]
```diff
@@ -69,7 +69,9 @@ program
     var file = program.args[0];
     var options = getOptions();
     options.watch = true;
-    createAndSave([file], options);
+    createAndSave([file], options, function() {
+
```
Comment (sudodoki) [inline bin/cleaver]: can you have it the same way, on single line, like in the other one, please?

**Comment by sudodoki** [comment]
Thanks so much for stepping up and providing fix for this! 👏  Can you do this little change from comment as well, please?

**Comment by kyuumeitai** [comment]
There!

**Comment by sudodoki** [comment]
Looks good. Will try to merge & publish over weekend or maybe someone else will get to that before me. 😉

**Comment by sudodoki** [comment]
Fixes https://github.com/jdan/cleaver/issues/171

**Pull request merged**